tests/core/test_mixer.py

```diff
@@ -33,3 +33,21 @@ def test_set_mute(self):
         self.core.mixer.set_mute(True)
 
         self.mixer.set_mute.assert_called_once_with(True)
+
+
+class CoreNoneMixerTest(unittest.TestCase):
+    def setUp(self):  # noqa: N802
+        self.core = core.Core(None, backends=[])
+
+    def test_get_volume(self):
+        self.assertEqual(self.core.mixer.get_volume(), -1)
+
+    def test_set_volume(self):
+        self.assertEqual(self.core.mixer.set_volume(30), False)
+
+    def test_get_mute(self):
+        self.assertEqual(self.core.mixer.get_mute(), False)
+
+    def test_set_mute(self):
+        self.core.mixer.set_mute(True)
+        self.assertEqual(self.core.mixer.set_mute(True), True)
```

Comment:
The test name should tell more about what behavior you expect here. Currently, it is only defined in the implementation, not in tests or docstrings.